doc: Bluetooth: move AutoPTS setup pages

jori-nordic opened the pull request:
In preparation of further cleanup commits.
Removes noise in the navigation pane in the rendered docs.

https://builds.zephyrproject.io/zephyr/pr/70656/docs/connectivity/bluetooth/bluetooth-qual.html

jori-nordic committed:
doc: Bluetooth: move AutoPTS setup pages

In preparation of further cleanup commits.
Removes noise in the navigation pane in the rendered docs.

Signed-off-by: Jonathan Rico <jonathan.rico@nordicsemi.no>

Thalley commented on doc/connectivity/bluetooth/bluetooth-qual.rst:
```diff
@@ -3,6 +3,21 @@
+Qualification setup
+*******************
+
+The Zephyr Bluetooth host is qualified using Bluetooth's PTS (Profile Tuning Suite) software.
```
```suggestion
The Zephyr Bluetooth host can be qualified using Bluetooth's PTS (Profile Tuning Suite) software.
```

It's been a long time since we've done a qualification, so might be a bit much to claim that is is qualified today.

Thalley replied:
True, but that is so long ago, that the current host and the qualified host are hardly the same host subsystem anymore :|

jori-nordic committed:
doc: Bluetooth: clarify qualification state

The BT subsystem has not been officially qualified for a somewhat long
time now. The code has diverged quite a bit from what it was.

Signed-off-by: Jonathan Rico <jonathan.rico@nordicsemi.no>
